### PR TITLE
fix(kit): use correct mask for `RU`-phone number

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes UIKIT-5911

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/kit/components/input-phone-international/input-phone-international.component.ts [hunk 1]
```diff
@@ -30,16 +29,15 @@ import {
...
-    TuiSizeL,
-    TuiSizeM,
-    TuiSizeS,
 } from '@taiga-ui/core';
 import {TuiCountryIsoCode} from '@taiga-ui/i18n';
-import {TUI_ARROW} from '@taiga-ui/kit/components/arrow';
+import {
+    TUI_INPUT_PHONE_INTERNATIONAL_OPTIONS,
+    TuiInputPhoneInternationalOptions,
+} from '@taiga-ui/kit/components';
```
Comment (nsbarsukov) [inline projects/kit/components/input-phone-international/input-phone-international.component.ts:37]: it breaks build

**Review by nsbarsukov — APPROVED** [review]

File: projects/kit/components/input-phone-international/input-phone-international.component.ts (hunk 1 above)
Comment (splincode) [reply projects/kit/components/input-phone-international/input-phone-international.component.ts]: fixed

**Review by waterplea — CHANGES_REQUESTED** [review]

File: projects/kit/components/input-phone-international/input-phone-international.options.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import {TuiSizeL, TuiSizeM, TuiSizeS} from '@taiga-ui/core';
+import {TuiCountryIsoCode} from '@taiga-ui/i18n';
+import {TUI_ARROW} from '@taiga-ui/kit/components/arrow';
+import {PolymorpheusContent} from '@tinkoff/ng-polymorpheus';
+
+export interface TuiInputPhoneInternationalOptions {
+    readonly countries: readonly TuiCountryIsoCode[];
+    readonly countryIsoCode: TuiCountryIsoCode;
+    readonly open: boolean;
+    readonly arrow: PolymorpheusContent<
```
[the remark this answers is not in the document]
Comment (splincode) [reply projects/kit/components/input-phone-international/input-phone-international.options.ts]: fixed

File: projects/kit/components/input-phone-international/input-phone-international.options.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+import {TuiContextWithImplicit} from '@taiga-ui/cdk';
+import {TuiSizeL, TuiSizeM, TuiSizeS} from '@taiga-ui/core';
+import {TuiCountryIsoCode} from '@taiga-ui/i18n';
+import {TUI_ARROW} from '@taiga-ui/kit/components/arrow';
+import {PolymorpheusContent} from '@tinkoff/ng-polymorpheus';
+
+export interface TuiInputPhoneInternationalOptions {
+    readonly countries: readonly TuiCountryIsoCode[];
+    readonly countryIsoCode: TuiCountryIsoCode;
+    readonly open: boolean;
```
[the remark this answers is not in the document]
Comment (splincode) [reply projects/kit/components/input-phone-international/input-phone-international.options.ts]: fixed

File: projects/kit/components/input-phone-international/input-phone-international.component.ts [hunk 4]
```diff
@@ -35,14 +35,18 @@ import {
     TuiSizeS,
 } from '@taiga-ui/core';
 import {TuiCountryIsoCode} from '@taiga-ui/i18n';
-import {TUI_ARROW} from '@taiga-ui/kit/components/arrow';
+import {TUI_ARROW} from '@taiga-ui/kit/components';
```
Comment (waterplea) [inline projects/kit/components/input-phone-international/input-phone-international.component.ts:38]: ⚠

File: projects/kit/components/input-phone/input-phone.options.ts [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    readonly phoneMaskAfterCountryCode: string;
+    readonly allowText: boolean;
+    readonly search: string;
+    readonly countryCode: string;
+}
+
+export const TUI_INPUT_PHONE_DEFAULT_OPTIONS: TuiInputPhoneOptions = {
+    phoneMaskAfterCountryCode: '### ###-##-##',
+    allowText: false,
+    search: '',
```
Comment (waterplea) [inline projects/kit/components/input-phone/input-phone.options.ts:13]: Let's remove this

File: projects/kit/components/input-phone-international/input-phone-international.component.ts (hunk 4 above)
Comment (splincode) [reply projects/kit/components/input-phone-international/input-phone-international.component.ts]: fixed

File: projects/kit/components/input-phone/input-phone.options.ts (hunk 5 above)
Comment (splincode) [reply projects/kit/components/input-phone/input-phone.options.ts]: fixed

**Review by waterplea — APPROVED** [review]

**Pull request merged**